static/js/brand-store/Members/MembersTable.js

```diff
@@ -1,8 +1,61 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import PropTypes from "prop-types";
 import { MainTable, CheckboxInput } from "@canonical/react-components";
 
-function MembersTable({ filteredMembers }) {
+function MembersTable({ filteredMembers, changedMembers, setChangedMembers }) {
+  const [members, setMembers] = useState(filteredMembers);
+
+  const handleRoleChange = (member, role) => {
+    const updatedMembers = members.map((item) => {
+      let updatedItem = {};
+
+      if (item.id === member.id) {
```

Comment:
To make the code easier to follow you can invert this conditional and return early. This will remove an indent and others won't have to try and figure out what happens if it's not true (in this case, nothing)

Reply:
Done